packages/core/src/lib/utils/providers.ts

```diff
@@ -75,12 +77,20 @@ function normalizeOAuth(
     c.redirectProxyUrl = `${c.redirectProxyUrl}/callback/${c.id}`
   }
 
+  let discoveredAs: o.AuthorizationServer | undefined
+  if (c.issuer) {
```

Comment:
- Not working if authorization server does not follow the specifications defined in OpenID Connect Discovery 1.0
- Customized wellKnown is not available